src/object-keys-entries.js

```diff
@@ -0,0 +1,39 @@
+export function sortedKeys(obj) {
+  return Object.keys(obj).sort((a, b) => {
+    return a.length - b.length;
+  });
+}
+
+export function getFilteredKey(obj) {
+  for (let key of Object.keys(obj)) {
```

Comment:
using a for of loop also works. a filter will return an array though. Object.keys makes the keys of an object into strings and put them into an array. You could chain a filter off of that. This does work though so nice job.